### ISSUE TITLE
ConceptMap in CapabilityStatement isn't valid

### ISSUE THREAD

**Issue by mstrotta** [omoponfhir/omoponfhir-omopv5-stu3-conceptmapping-server#5]
We're using GT-FHIR2 for OMOPv5 in CS6440 at https://apps.hdap.gatech.edu/gt-fhir/fhir/metadata. The resource at `.rest[0].resource[0]` in the conformance statement looks like:
```
{ "type": "ConceptMap",
 "profile": {
 "reference": "http://hl7.org/fhir/StructureDefinition/ConceptMap"
 } } 
```
This doesn't appear to be conformant. For example, it doesn't contain an `interaction.code`. I'm using `fhirclient.py` to connect, and the client assumes that all resources `rest` have an `interaction`.

**Comment by myungchoi** [comment]
Hi,

That is because the resource ConceptMap in omoponfhir server current does not support any resource level interactions. We use this resource to support other operation, translation between code systems. And, the operation needs to go through ConceptMap resource. In the future, we would provide the ConceptMap from concept table of OMOP. But, currently, we do not have it. 

The spec says that if we have a resource, we need to have the interaction with at least one interaction. But, we don’t have any implemented. So, it has no interaction. Is it hard for you to check if the key exists? 

Thanks,
Myung

**Comment by myungchoi** [comment]
Actually, I see your issue. You are using smart on FHIR python library.

I will see if I can fix it.

**Comment by mstrotta** [comment]
Yes. The quick way to prevent an error in the Python library was to change
the `CapabilityStatementRestResource` domain model so that `interaction` is
not required. That worked for me locally, but I thought it would be good to
raise the issue.

Best,
Mike

**Comment by myungchoi** [comment]
The missing interaction is fixed. You should be able to use smart on FHIR python library without changing it.

Please note that the ConceptMap will always return null.

Thanks,
